src/main/java/com/twitter/whiskey/net/SpdySession.java

```diff
@@ -670,7 +670,7 @@ private void sendData(SpdyStream stream) {
     private void sendClientSettings() {
 
         SpdySettings settings = new SpdySettings();
-        settings.setValue(SpdySettings.MAX_CONCURRENT_STREAMS, 100);
+        settings.setValue(SpdySettings.MAX_CONCURRENT_STREAMS, localMaxConcurrentStreams);
```

Comment:
yes, just verified that a simple get with no pushes works if the default is zero.